array_api_strict/tests/test_elementwise_functions.py

```diff
@@ -91,12 +99,57 @@ def nargs(func):
     "trunc": "real numeric",
 }
 
+
+def test_nargs():
```

Comment:
A short test to make sure `nargs` works but also that all of the functions that we look at have "the right signature" - I found `logaddexp` was missing that trailing `/` when working on `nargs`. So it seems useful to have a "all functions have a reasonable number of arguments" test.

Reply:
That's good. The array-api test suite doesn't check for positional-only, though it probably could now that it uses inspect.signature.